### PR TITLE
Improve input to correctly read operational HRRR/RAP. 

### PR DESCRIPTION
**Pull request by hu5970**
The following records have changed the name or location number for the following variables:
 ice mixing ratio, 
vegetation fraction, 
vegmax, 
vegmin, 
LAI

The input_data.F90 is updated to read in those fields.

The code has been tested with both RAPx/HRRRx and RAP/HRRR operation (combined) grib2 files.

### CODE REVIEW

File: sorc/chgres_cube.fd/input_data.F90 [hunk 1]
```diff
@@ -5050,6 +5060,11 @@ subroutine read_input_sfc_grib2_file(localpet)
 
      vname=":VEG:"
      rc= grb2_inq(the_file, inv_file, vname,slev,'n=1107:',data2=dummy2d)
```
Comment (JeffBeck-NOAA) [inline sorc/chgres_cube.fd/input_data.F90:5062]: @hu5970, I see unique identifiers are now being used for each of these fields (var2_0_232 for max vegetation fraction and :LAI: for LAI).  So, are we able to identify these fields directly without using the record order number?  What changed in the grib2 files?

I also don't see a change for vegetation fraction (only min/max veg frac, and LAI), is that correct?

Comment (hu5970) [reply sorc/chgres_cube.fd/input_data.F90:5062]: Jeff,

Yes, the operational HRRR grib2 files has only one VEG:surface.  The chgres_cube can read it correctly. 
The max and min veg fraction now can be identified by its unique name (var2_0_232) without record number. 

Grib2 table used by RAPx/HRRRx is different from operational one, which has been updated and reviewed by NCO before operation.

**Comment by hu5970** [comment]
@jaymes-kenyon @christinaholtNOAA @chunhuazhou 

I just merged the changes in chgres_cube for reading operational HRRR grib2 files. Please update RRFS_dev4/chgres_cube to use this branch. It should works with RAPx and HRRRx also and better to test it before switch to operational RAP/HRRR.

Thanks,
Ming

**Comment by jaymes-kenyon** [comment]
@hu5970 — my [PR for Externals.cfg](https://github.com/NOAA-GSL/ufs-srweather-app/pull/15) just needs approval, and I can get your update to chgres_cube deployed into RRFS-dev4

**Pull request merged**